app/code/Transoft/Blog/view/frontend/templates/blogListTmpl.phtml

```diff
@@ -0,0 +1,23 @@
+<?php
+use Transoft\Blog\Block\Bloglist;
+?>
+<?php /** @var Bloglist $block*/
+?>
+<?php
+$bloglist = $block->getData('bloglist');
```

Comment:
Current template should be assigned to Bloglist

Reply:
fixed